llvm/lib/Analysis/LoopAccessAnalysis.cpp

```diff
@@ -2127,14 +2129,15 @@ MemoryDepChecker::isDependent(const MemAccessInfo &A, unsigned AIdx,
     return Dependence::Forward;
   }
 
-  int64_t MinDistance = SE.getSignedRangeMin(Dist).getSExtValue();
+  std::optional<int64_t> MinDistance =
+      SE.getSignedRangeMin(Dist).trySExtValue();
```

Comment:
Same reasoning as for ConstDist should also apply, could retain `getSExtValue()`?